### PR TITLE
Add Constant for Gender.

### PR DESCRIPTION
**Pull request by Emistry**
Improve readability of scripts when using variable `Sex`.

### CODE REVIEW

**Comment by dastgir** [comment]
MALE / FEMALE are too generic name..
Maybe have some different name?
@MishimaHaruna

**Comment by 4144** [comment]
Maybe G_MALE/G_FEMALE?

**Comment by MishimaHaruna** [comment]
I'd suggest `SEX_MALE` / `SEX_FEMALE` (bonus: it even matches the enum we have in mmo.h)

**Comment by EPuncker** [comment]
update script_commands as well :+1:

**Comment by Emistry** [comment]
@EPuncker  any suggestion for the template or structure ? https://github.com/HerculesWS/Hercules/blob/master/doc/script_commands.txt#L555
```
Sex         - 0 if female, 1 if male.
			  Additionally, you could also use constants: SEX_MALE or SEX_FEMALE to match the gender.
```
or 
```
Sex         - return types:
				0 - SEX_FEMALE		- Female Character
				1 - SEX_MALE		- Male Character
```

**Comment by dastgir** [comment]
If it matches the source , why don't we move this constant to source? So these constants will always be in sync?

**Comment by Mysteries** [comment]
I agree to add the constants of SEX_MALE and SEX_FEMALE. Looks better / seems better xD

File: db/constants.conf [hunk 1]
```diff
@@ -3806,4 +3806,9 @@ constants_db: {
 	C_CHOCOLATE: 0xD2691E
 	C_GOLD:      0xFFD700
 	C_VIOLET:    0xEE82EE
+	
+	/** Gender */
+	FEMALE:	0
```
Comment (MishimaHaruna) [inline db/constants.conf]: Still not renamed here

File: doc/script_commands.txt [hunk 2]
```diff
@@ -553,6 +553,7 @@ Weight      - Amount of weight the character currently carries.
 MaxWeight   - Maximum weight the character can carry.
               Display as in MaxWeight/10.
 Sex         - 0 if female, 1 if male.
```
Comment (dastgir) [inline doc/script_commands.txt]: Encourage constant and remove numbers from documentation?

**Comment by dastgir** [comment]
Also, a better commit message

**Comment by Emistry** [comment]
> Also, a better commit message

hmm... Example ?

**Comment by MishimaHaruna** [comment]
Taking care of this - I'll rebase, split in commits, merge.

**Comment by MishimaHaruna** [comment]
Merged via 12a85a011c2008ea92b9197e79646aaa6ece5a41

**Pull request closed without merging**